db/migrate/20180109073625_add_current_sign_in_ip_to_admins.rb

```diff
@@ -0,0 +1,5 @@
+class AddCurrentSignInIpToAdmins < ActiveRecord::Migration[5.0]
+  def change
+    add_column :admins, :current_sign_in_ip, :inet
```

Comment:
same as above

Reply:
this is requirement from devise's :trackable

Reply:
did you run `rails generate devise MODEL`? It should generate the model for you with all required columns, we never have to do it manually

Reply:
yes, I ran `rails generate devise Admin`, and it generate model with `last_sign_in_at` and `last_sign_in_ip` but `current_sign_in_at` and `current_sign_in_ip` are missing. ActiveRecord complain when I try to save an Admin object, that's why I have to create those 2 migrations. Not sure if I missed something